Error reading blackrock nev file, IndexError: too many indices, in blackrockrawio.py - due to deprecation drop in Numpy 1.23.0

To replicate:
`brIO = neo.io.BlackrockIO('datafile002.nev')`

Error is at:
```
                    mask = [ev_ids == i]
>                   curr_data = data[mask]
E                   IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed

../../../../miniconda38/envs/mh38a/lib/python3.8/site-packages/neo/rawio/blackrockrawio.py:442: IndexError
```

Datafile:

[datafile002.nev.gz](https://github.com/NeuralEnsemble/python-neo/files/9329535/datafile002.nev.gz)

--- a/neo/rawio/blackrockrawio.py
+++ b/neo/rawio/blackrockrawio.py
@@ -438,8 +438,7 @@ class BlackrockRawIO(BaseRawIO):
             # Find maximal and minimal time for each nev segment
             for k, (data, ev_ids) in self.nev_data.items():
                 for i in np.unique(ev_ids):
-                    mask = [ev_ids == i]
-                    curr_data = data[mask]
+                    curr_data = data[ev_ids == i]
                     if curr_data.size > 0:
                         if max(curr_data['timestamp']) >= max_nev_times.get(i, 0):
                             max_nev_times[i] = max(curr_data['timestamp'])

